_config.yml

```diff
@@ -27,17 +27,48 @@ markdown:    kramdown
 include:
 - _static
 
+# TODO use variables instead of 1.21
```

Comment:
Drop this please, variables in config.yml cannot be used according to the documentation. We used to have it here anyway (see `1.20` on deleted like 34) and let's not try to solve this problem here.